README.md

```diff
@@ -0,0 +1,230 @@
+# cuda_memtest
+
+This software tests GPU memory for hardware errors and soft errors using CUDA (or OpenCL).
+
+## Note for this Fork
+
+This is a fork of the original, yet long-time unmaintained project at https://sourceforge.net/projects/cudagpumemtest/ .
+
+After our fork in 2013 (v1.2.3), we primarily focused on support for newer CUDA versions and support of newer Nvidia hardware.
+Pull-requests maintaining the OpenCL versions are nevertheless still welcome.
+
+## License
+
+Illinois Open Source License
+
+University of Illinois/NCSA  
+Open Source License
+
+Copyright 2009-2012,    University of Illinois.  All rights reserved.  
+Copyright 2013-2019,    The developers of PIConGPU at Helmholtz-Zentrum Dresden-Rossendorf
+
+Developed by:
+
+  Innovative Systems Lab  
+  National Center for Supercomputing Applications  
+  http://www.ncsa.uiuc.edu/AboutUs/Directorates/ISL.html
+
+Forked and maintained for newer Nvidia GPUs since 2013 by:
+
+  Axel Huebl and Rene Widera  
+  Computational Radiation Physics Group  
+  Helmholtz-Zentrum Dresden-Rossendorf  
+  https://www.hzdr.de/crp
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of 
+this software and associated documentation files (the "Software"), to deal with 
+the Software without restriction, including without limitation the rights to use,
+copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the 
+Software, and to permit persons to whom the Software is furnished to do so, subject
+to the following conditions:
+
+* Redistributions of source code must retain the above copyright notice, this list 
+  of conditions and the following disclaimers.
+
+* Redistributions in binary form must reproduce the above copyright notice, this list
+  of conditions and the following disclaimers in the documentation and/or other materials
+  provided with the distribution.
+
+* Neither the names of the Innovative Systems Lab, the National Center for Supercomputing
+  Applications, nor the names of its contributors may be used to endorse or promote products
+  derived from this Software without specific prior written permission.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
+INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE CONTRIBUTORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT 
+OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+DEALINGS WITH THE SOFTWARE.
+
+## Compile and Run
+
+### Compile
+
+Inside the source directory, run:
```

Comment:
Don't we normally recommend an out-of-source build? Guess for this in-source should be fine when done like described here.

Reply:
Running in a quick-&-dirty empty dir is not perfect but good enough.